src/Http/ServerRequestFactory.php

```diff
@@ -183,152 +180,4 @@ public function createServerRequest(string $method, $uri, array $serverParams =
 
         return new ServerRequest($options);
     }
-
-    /**
-     * Create a new Uri instance from the provided server data.
-     *
-     * @param array $server Array of server data to build the Uri from.
-     *   $_SERVER will be added into the $server parameter.
-     * @return array
-     * @psalm-return array{0: \Psr\Http\Message\UriInterface, 1: string, 2: string}
-     */
-    public static function createUri(array $server = []): array
-    {
```

Comment:
This will be tricky to add a deprecation for unless we backport the http-factory. I could take care of the backport.

Reply:
The chances of this method method being used outside of the core are very slim IMO. Except for perhaps tests there would be rarely a reason to create an Uri instance yourself.

Reply:
That's fair. Lets go without a deprecation and see how it goes :smile: